brion/morphology.h

```diff
@@ -95,10 +108,21 @@ class Morphology : public boost::noncopyable
      * @param overwrite true to allow overwrite of existing file
      * @throw std::runtime_error if file could not be created
      * @version 1.0
+     * @deprecated
      */
     BRION_API Morphology( const std::string& target,
                           MorphologyVersion version, bool overwrite = false );
```

Comment:
I've realized that if we remove this constructor, we won't be able to write SWC morphologies in the future. Not that it really matters, as the writing functions in the SWC plugin are unimplemented, but just to make you aware of the fact.
We can keep the function deprecated to discourage usage until we have a clearer idea.